src/examples/draggable/DropzoneDanger.tsx

```diff
@@ -0,0 +1,254 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { RefObject, forwardRef, useCallback, useEffect, useRef, useState } from 'react';
+import styled, { useTheme } from 'styled-components';
+import {
+  Announcements,
+  MeasuringStrategy,
+  DndContext,
+  useSensors,
+  useSensor,
+  MouseSensor,
+  TouchSensor,
+  KeyboardSensor,
+  KeyboardCoordinateGetter,
+  DragOverlay,
+  DragStartEvent,
+  DragOverEvent,
+  DragEndEvent,
+  useDraggable,
+  useDroppable
+} from '@dnd-kit/core';
```

Comment:
`@dnd-kit` needs to be added to https://github.com/zendeskgarden/website/blob/eec7adaf8c518f0a2e8f5dc6d46b153ee32231e6/src/components/MarkdownProvider/components/CodeExample/utils/retrieveCodesandboxParameters.ts#L52